documentation/modules/ref-prometheus-metrics.adoc

```diff
@@ -5,17 +5,18 @@
 [id='ref-metrics-{context}']
 = Metrics configuration
 
-Prometheus metrics are enabled by configuring the `metrics` property in following resources:
+Prometheus metrics are enabled by configuring the `metricsConfig` property in following resources:
 
 * `Kafka.spec.kafka`
 * `Kafka.spec.zookeeper`
 * `KafkaConnect.spec`
 * `KafkaConnectS2I.spec`
 
-When the `metrics` property is not defined in the resource, the Prometheus metrics will be disabled.
-To enable Prometheus metrics export without any further configuration, you can set it to an empty object (`{}`).
+When the `metricsConfig` property is not defined in the resource, the Prometheus metrics are disabled.
+To enable Prometheus metrics export without further configuration, you can reference a ConfigMap containing empty file under a specified key.
+All metrics without any renaming will be exposed in such a case.
 
-.Example of enabling metrics without any further configuration
+.Enabling metrics without further configuration
```

Comment:
Not sure this example still makes sense as the example here is the same as the example right below it. So I guess you should either remove one of the examples. Or you should add also a YAML example of the config map => here one with config mp having just empty file under the key and below an example of a config map with some configuration.